Content.Server/StationEvents/Components/SlimesSpawnRuleComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Content.Server.StationEvents.Events;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent, Access(typeof(SlimesSpawnRule))]
+public sealed class SlimesSpawnRuleComponent : Component
+{
+	[DataField("SpawnedPrototypeChoices")]
+    public List<string> SpawnedPrototypeChoices = new()
+        {"MobAdultSlimesBlueAngry", "MobAdultSlimesGreenAngry", "MobAdultSlimesYellowAngry"};
+}
```

Comment:
1. This should be made generic with the other spawn rules.
2. You mixed tabs and spaces.
3. Needs typeserializer for prototypes.

Reply:
corrected